modules/cluster-logging-release-notes-5.0.8.adoc

```diff
@@ -0,0 +1,11 @@
+[id="cluster-logging-release-notes-5-0-8"]
+= OpenShift Logging 5.0.8
+
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3526[RHBA-2021:3526 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.8)].
+
+[id="openshift-logging-5-0-8-bug-fixes"]
+== Bug fixes
+
+This release also includes the following bug fixes:
+
+* link:https://issues.redhat.com/browse/LOG-1741[LOG-1741] - ElasticSearch Operator isn't upgraded due to wrong olm.skipRange
```

Comment:
We need to elaborate this bug fix to match the OCP descriptions for bug fixes: https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-bug-fixes and use the RN text provided by the Devs in the Jira: Due to an issue in the release pipeline scripts, the value of the `olm.skipRange` field remained unchanged at `5.2.0` and was not updated when the z-stream number, `0`, increased. The current release fixes the pipeline scripts to update the value of this field when the release numbers change.